install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml

```diff
@@ -0,0 +1,37 @@
+MultiBeamSEM: {
+    class: Microscope,
+    role: mbsem,
+    children: ["ASM", "EBeamScanner", "MirrorDescanner", "MPPC",],
+}
+
+"ASM": {
+  class: technolution.AcquisitionServer,
+  role: "main",
+  init: {
+    # Use following host url for real system instead of simulator: "http://192.168.10.10:8080/v2"
+    host: "http://localhost:8080/v2",
+    externalStorage: { "host": "localhost",
+                        "username": "username",
+                        "password": "password",
+                        "directory": "image_dir" }
+  },
+  children: { "EBeamScanner" : "EBeamScanner" ,
+              "MirrorDescanner"  : "MirrorDescanner" ,
+              "MPPC" : "MPPC"
+  },
+}
+
+"EBeamScanner": {
+  # Internal child of ASM, so no class
+  role: "multibeam",
+}
+
+"MirrorDescanner": {
+  # Internal child of ASM, so no class
+  role: "galvo",
```

Comment:
"galvo" does not look like a good name to me. The corresponding child is named "MirrorDescanner". It is not obvious that the mirror descanner is a mirror galvonometer. I suggest some roles that are inspired by the class names as we usually stick with them "forever". "mppc" and "multibeam" are fine, for the galvo I suggest maybe "multibeam-descanner" or just "descanner" to keep it short?

Reply:
Let's go for descanner then.